### PR TITLE
ENG-6511 & ENG-6509 & ENG-6489 - MIHR dependency update, Configure Update, Start Update

### PR DESCRIPTION
**Pull request by kevinsites**

### CODE REVIEW

**Review by priya-xavier — DISMISSED** [review]

File: SDKTest/NeuroIDClassTests.swift [hunk 1]
```diff
@@ -629,6 +673,30 @@ class NIDClientSiteIdTests: XCTestCase {
 
         assert(NeuroID.siteId == "test_site")
     }
+
```
Comment (kishisaka-nid) [inline SDKTest/NeuroIDClassTests.swift:676]: nice!

**Review by kishisaka-nid — DISMISSED** [review]
LGTM. 🚢

File: SDKTest/NeuroIDClassTests.swift [hunk 2]
```diff
@@ -97,6 +97,7 @@ class NeuroIDClassTests: XCTestCase {
     }
 
     func test_start_failure() {
+        tearDown()
```
Comment (kishisaka-nid) [inline SDKTest/NeuroIDClassTests.swift:100]: rather important!

**Review by kishisaka-nid — COMMENTED** [review]
Good to go! 🚢

**Review by kishisaka-nid — APPROVED** [review]

**Pull request merged**